### ISSUE TITLE
API returns nothing for valid ID 

### ISSUE THREAD

**Issue by bernt-matthias** [bio-tools/biotoolsRegistry#604]
`mummer4` appears to be a valid entry:  https://bio.tools/mumer4, but the API does not find anything for it: https://bio.tools/api/tool/mummer4/?format=json

**Comment by anitamnd** [comment]
Hi! It seems like the ID is mispelled in the API call - mummer instead of mumer: 

https://bio.tools/api/tool/mumer4/?format=json

**Comment by bernt-matthias** [comment]
True. Difficult to spot, since the software is spelled with 2 `m` :). Thanks for letting me know.
